rq/worker.py

```diff
@@ -678,6 +732,11 @@ def move_to_failed_queue(self, job, *exc_info):
         self.log.warning('Moving job to {0!r} queue'.format(self.failed_queue.name))
         self.failed_queue.quarantine(job, exc_info=exc_string)
 
+    def move_to_failed_queue_unhandled(self, job, message):
```

Comment:
Since this method is only used in one place, let's just place the code inline in `monitor_work_horse()`